yt_dlp/extractor/youtube.py

```diff
@@ -5084,10 +5089,35 @@ def _playlist_entries(self, video_list_renderer):
                 continue
             yield self._extract_video(renderer)
 
+    def _extract_lookup_view_model(self, renderer):
+        content_id = renderer.get('contentId')
+        if not content_id:
+            return
+        entry_url = None
+        if renderer.get('contentType') in ('LOCKUP_CONTENT_TYPE_PLAYLIST', 'LOCKUP_CONTENT_TYPE_PODCAST'):
+            entry_url = f'https://www.youtube.com/playlist?list={content_id}'
+        else:
+            self.report_warning(
+                f'Unsupported lookup view model content type "{renderer.get("contentType")}"{bug_reports_message()}', only_once=True)
+            return
+        return self.url_result(
+            entry_url,
+            ie=YoutubeTabIE, video_id=content_id,
+            **traverse_obj(renderer, {
+                'title': ('metadata', 'lockupMetadataViewModel', 'title', 'content', {str}),
+            }),
```

Comment:
this formatting is kinda odd
```suggestion
            entry_url, ie=YoutubeTabIE, video_id=content_id, title=traverse_obj(renderer, (
                'metadata', 'lockupMetadataViewModel', 'title', 'content', {str})),
```